build-tools/build-indexes

```diff
@@ -1017,7 +1016,216 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
 			}
 		}
 	}
-
+	//Scoopapa code
+	console.log('');
+	BattleTeambuilderTable.ClientMods = {};
+	for (const modid of Object.keys(ClientMods)) {
+		try { //catch loading errors so the whole thing doesn't break if one mod does
+			//Gather data about formats
+			BattleTeambuilderTable.ClientMods[modid] = {};
+			const modDex = Dex.mod(modid);
+			const modData = modDex.data;
+			let modGen = 0;
+			BattleTeambuilderTable.ClientMods[modid].formats = [];
+			for (let formatid in Dex.data.Formats) { //Find formats using this mod
+				let format = Dex.getFormat(formatid);
+				if (format.mod === modid) {
+					if (formatid.slice(0, 3) === 'gen' && formatid.charAt(3) !== '8') modGen = (formatid.slice(0, 4));
+					BattleTeambuilderTable.ClientMods[modid].formats.push(format.fullname);
+				}
+			}
+			const standardTiers = ['uber', 'ou', 'uubl', 'uu', 'rubl', 'ru', 'nubl', 'nu', 'publ',
+				'pu', 'zu', 'nfe', 'lcuber', 'lc', 'cap', 'caplc', 'capnfe', 'ag', 'duber', 'dou',
+				'dbl', 'duu', 'dnu', 'illegal', 'unreleased'];
+			if (ClientMods[modid].customTiers) BattleTeambuilderTable.ClientMods[modid].customTiers = ClientMods[modid].customTiers;
+			else BattleTeambuilderTable.ClientMods[modid].customTiers = [];
+			for (const speciesid in modData.FormatsData) { //Find any nonstandard tiers used by this mod
+				const tier = modData.FormatsData[speciesid].tier;
+				if (typeof tier === 'undefined') continue;
+				if (!standardTiers.includes(toID(tier)) &&
+					!BattleTeambuilderTable.ClientMods[modid].customTiers.includes(tier)) {
+					BattleTeambuilderTable.ClientMods[modid].customTiers.push(tier);
+				}
+			}
```

Comment:
```suggestion
			// Find any nonstandard tiers used by this mod
			for (const speciesid in modData.FormatsData) {
				const tier = modData.FormatsData[speciesid].tier;
				if (tier === undefined) continue;
				if (!standardTiers.includes(toID(tier)) &&
					!BattleTeambuilderTable.ClientMods[modid].customTiers.includes(tier)) {
					BattleTeambuilderTable.ClientMods[modid].customTiers.push(tier);
				}
			}
```

Reply:
Although tbh it's probably easiest to just check `tier &&` in the second `if` statement